tests/api/test_serve.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from lightly.api import serve
+
+
+def test__translate_path(tmp_path: Path) -> None:
+    tmp_file = tmp_path / "hello/world.txt"
+    assert serve._translate_path(path="/hello/world.txt", directories=[]) == ""
+    assert serve._translate_path(path="/hello/world.txt", directories=[tmp_path]) == ""
+    tmp_file.mkdir(parents=True, exist_ok=True)
```

Comment:
do we want to clean this directory up after the test or is this python magic and "inmemory" somehow?

Reply:
It's a temporary directory which is removed automatically.